libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmOpenShiftVersionSelect.tsx

```diff
@@ -57,6 +59,8 @@ type OcmOpenShiftVersionSelectProps = {
 };
 const OcmOpenShiftVersionSelect = ({ versions }: OcmOpenShiftVersionSelectProps) => {
   const { t } = useTranslation();
+  const [selectedValue, setSelectedValue] = React.useState<OpenshiftVersionOptionType>(); // Estado para almacenar el valor seleccionado
```

Comment:
Wouldn't it be better if you set the value on the form, rather than having to keep the value somewhere else?